technologies/FlexBE/README.md

```diff
@@ -89,6 +90,7 @@ The `pyrobosim_flexbe_states` package under `technologies/FlexBE` includes:
 * `pick_action_state.py` - Does pick action at current location
 * `place_action_state.py` - Does place action at current location
 * `plan_path_state.py` - Request plan from current location to target location
+* `run_btcpp_tree_state.py` - Run a behavior tree using custom BehaviorTree.CPP v4 executor
```

Comment:
IMO not needed to specify `v4`, as the convention is that is the latest, and if you're on the older versions you specify it.